### ISSUE TITLE
Android: Allow to set display name per call

### ISSUE THREAD

**Issue by rumwerfer** [jitsi/jitsi-meet#6062]
**Is your feature request related to a problem you are facing?**
When switching between calls of different organizations / different contexts, people will sometimes recognize me under different names (e.g. last name, first name, nickname).
In a browser, that works well for me: My display name is saved individually per call.
On Android: To change my display name,  I have to go to the settings, before starting the call.

**Describe the solution you'd like**
I'd like to be able to set the display name for each call individually on Android.

**Comment by saghul** [comment]
Nott sure. how you're doing it for web,  but I'm pretty sure we don't store your name per call.

**Comment by blu-IT** [comment]
> **Describe the solution you'd like**
> I'd like to be able to set the display name for each call individually on Android.

Well, I think **this is the advantage** of jitsi-meet app for Android or Desktop, that you can set your personal settings and they **stay saved**, that you don't have to set them everytime again and again.

Regards

**Comment by rumwerfer** [comment]
True. But the possibility to save them for each call individually wouldn’t necessarily spoil that.
I thought that you kinda have a default setting that is applied to all calls, until you change the setting for a call specifically.

Maybe my use case is quite rare.

**Comment by rumwerfer** [comment]
@saghul the calls are all on different instances, if that explains

**Comment by soxhi8** [comment]
The option would be very much welcome. I need different names per call, but there is no option prior to or in the call itself, using the app on Android.
